pandas/tests/frame/test_constructors.py

```diff
@@ -2530,6 +2530,19 @@ def test_from_2d_object_array_of_periods_or_intervals(self):
         expected = DataFrame({0: pi, 1: ii, 2: pi, 3: ii})
         tm.assert_frame_equal(df3, expected)
 
+    @pytest.mark.parametrize(
+        "col_a, col_b",
+        [
+            ([[1], [2]], np.array([[1], [2]])),
+            (np.array([[1], [2]]), [[1], [2]]),
+            (np.array([[1], [2]]), np.array([[1], [2]])),
+        ],
+    )
+    def test_error_from_2darray(self, col_a, col_b):
+        msg = "Data must be 1-dimensional"
```

Comment:
probably need to update the message here?

Reply:
My bad. Updated the msg in test.